tailwind.config.ts

```diff
@@ -41,6 +41,9 @@ export default {
 			},
 			maxWidth: {
 				"large-screen": "120rem"
+			},
+			transitionProperty: {
+				height: "height"
```

Comment:
I didn't know this was a thing! Can `transition-[height]` work without us adding a property here?

Reply:
Update: you [can](https://tailwindcss.com/docs/transition-property#arbitrary-values) (exact same example)! I think it's better to do so for one-time use, what do you think?